docs/source/reference.adoc

```diff
@@ -360,3 +360,75 @@ because generating a new token is an expensive operation. Get the access token i
 curl -H "Content-Type: application/json" -X POST -d '{"refresh_token":"<REFRESH_TOKEN>"}' https://auth.openshift.io/api/token/refresh
 
 As a result, you will get a JSON with an access token which you can use in your test.
+
+== Token validation
+
+=== The JWT contents
+
+The token is likely to contain the following claims.
+
+[source]
+ "acr": "0",
```

Comment:
Add the header of the token too. Not the body only. "kid" is in the header.

Reply:
Done.